checks/mysql/check_test.go

```diff
@@ -16,7 +21,45 @@ func TestNew(t *testing.T) {
 		DSN: os.Getenv(mysqlDSNEnv),
 	})
 
-	if err := check(); err != nil {
-		t.Fatalf("MySQL check failed: %s", err.Error())
+	err := check()
+	require.NoError(t, err)
+}
+
+func TestEnsureConnectionIsClosed(t *testing.T) {
+	if os.Getenv(mysqlDSNEnv) == "" {
```

Comment:
Would it not be nicer to use something like https://github.com/DATA-DOG/go-sqlmock to mock the db?

Reply:
I do not want to mock the DB, I want to run "integration" test against real backends - all are running in containers.

Reply:
Ok just a question but why not move this to a separate file and use a build flag/tag so it can be run explicitly? That way the test can fail if the env var is not set and this will not become some abandoned test that always skips

Reply:
Makes sense. But I just made them all required instead of intrucing flag/tag.